.travis.yml

```diff
@@ -37,14 +38,16 @@ matrix:
             - ubuntu-toolchain-r-test
           packages:
             - g++-4.9
+            - libboost-all-dev
```

Comment:
The shoutcast feature will leverage **Boost** during unit test to write files to the `/tmp` directory.  This helps test the output of `config.xml` file for settings.

Reply:
I dont think we want a boost dependency just for the unit tests!

Reply:
I agree, was kind of on the fence with the inclusion of boost, as it’s a pretty heavy weight library. 

I will remove, and rework the shoutcast tests.